pgs-mock/Dockerfile.pgs-mock

```diff
@@ -1,9 +0,0 @@
-FROM node:18.19.0-alpine
```

Comment:
delete pagopa-pgs-mock from docker-compose.yaml file too, as for now docker compose build is broken for missing still referenced Dockerfile